cirq-core/cirq/transformers/transformer_api.py

```diff
@@ -274,14 +293,13 @@ def transformer(cls_or_func: Any) -> Any:
         method = cls.__call__
 
         @functools.wraps(method)
-        def method_with_logging(
-            self, circuit: 'cirq.AbstractCircuit', context: TransformerContext
-        ) -> 'cirq.AbstractCircuit':
+        def method_with_logging(self, circuit, **kwargs) -> 'cirq.AbstractCircuit':
```

Comment:
nit: could add back the `cirq.AbstractCircuit` annotation on `circuit`.

Reply:
done